virtual/lapacke/lapacke-3.8.ebuild

```diff
@@ -9,6 +9,9 @@ KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~hppa ~ia64 ~ppc ~ppc64 ~s390 ~sparc ~x86 ~a
 IUSE="eselect-ldso"
 
 RDEPEND="
-	>=sci-libs/lapack-3.8.0[lapacke,eselect-ldso?]
+	|| (
+		>=sci-libs/lapack-3.8.0[lapacke,eselect-ldso?]
+		sci-libs/openblas[eselect-ldso?]
```

Comment:
Can't do this before openblas is actually fixed and has `lapacke`, and even in that case we would need to make it dependent only on latest version, by adding a minimum dependency requirement.

Fair warning, there are a large number of changes needed before this PR can even be reviewed, so this PR will be waiting for quite a while.

Reply:
Good thing I'm not a proxy maintainer for opencv, and I'm merely providing a potential solution for the aforementioned bugs, whether it will be adopted is not really my concern.

Reply:
Thanks for the help though, we really need it :+1: .

Reply:
By the way are you going to bump the latest openblas to 0.3.11? Maybe you can take the opportunity to look into lapacke support for openblas. So far I know that v0.3.10 does install lapacke.h into its include directory.

Reply:
I have a lot of backlog of other fixes to do as well.
This is another on the list.
Adding new features (though important) has less priority than fixing packages.

Reply:
Thoughts on any workarounds until we can fix openblas?